### PR TITLE
chore!: Normalize repository, dropping node <10.13 support

### PR DESCRIPTION
**Pull request by sttk**
This PR updates this project:

- Updates dependency versions
- Introduces GitHub Actions
- Adds, changes and deletes some project files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,24 +18,29 @@
...
+    "reporter": [
+      "lcov",
+      "text-summary"
+    ]
+  },
+  "prettier": {
+    "singleQuote": true
   },
   "keywords": [
     "async",
```
Comment (phated) [inline package.json:46]: Can you also complete #13 by adding:
```suggestion
    "async",
    "parallel",
    "series",
    "each",
```

**Review by phated — CHANGES_REQUESTED** [review]
Just a couple things

**Comment by sttk** [comment]
@phated I've commited your suggestions.

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Blaine Bublitz, Eric Schoffstall and other contributors
+Copyright (c) 2014, 2021 Blaine Bublitz <blaine.bublitz@gmail.com> and Eric Schoffstall <yo@contra.io>
```
Comment (phated) [inline LICENSE:3]:
```suggestion
Copyright (c) 2014, 2021-2022 Blaine Bublitz <blaine.bublitz@gmail.com> and Eric Schoffstall <yo@contra.io>
```

**Review by phated — APPROVED** [review]
Looks great! Thanks @sttk 🍻

**Pull request merged**